site/ko/guide/distributed_training.ipynb

```diff
@@ -71,7 +71,7 @@
       "source": [
         "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
         "  <td>\n",
-        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/guide/distributed_training\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />TensorFlow.org에서 보기</a>\n",
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/distribute/keras\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />TensorFlow.org에서 보기</a>\n",
```

Comment:
This is not correct. Please revert.

Also, please submit translation changes as a separate pull request so the community can review.

Reply:
I change correct

I don't know last sentence how can i do, can you explain it more easily?

Reply:
I see the "The en/ directory is the source-of-truth." in "site/README.md"
I'm sorry I didn't check the document properly.